### PR TITLE
Update searching-array-methods.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Update incorrect information

If no elements match then *undefined* will be returned:

to:

If no elements match then `-1` will be returned:

and
//returns Alex
to:
//returns { name: 'Alex', age: 36 }

### CODE REVIEW

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: javascript/ecmascript-2015/array-updates/searching-array-methods.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ var people = [
 people.find(function(e, i, src) {
   return e.age >= 18;
 });
-//returns Alex
+//returns { name: 'Alex', age: 36 }
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/array-updates/searching-array-methods.md:54]:
```suggestion
// returns { name: 'Alex', age: 36 }
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
@Stefan-Stojanovic 

we should also:
- replace `var` with `let` in the content, pq, and rq sections
- replace the `ponyfoo.com` resource title with `ES6 Array Extensions in Depth`
- delete the `inAlgoPools: false` line since it's not used anymore

File: javascript/ecmascript-2015/array-updates/searching-array-methods.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply javascript/ecmascript-2015/array-updates/searching-array-methods.md]: Done

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by Stefan-Stojanovic** [comment]
> @Stefan-Stojanovic
> 
> we should also:
> 
> * replace `var` with `let` in the content, pq, and rq sections
> * replace the `ponyfoo.com` resource title with `ES6 Array Extensions in Depth`
> * delete the `inAlgoPools: false` line since it's not used anymore

done

**Review by kapnobatai137 — DISMISSED** [review]
:muscle:

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**